bin/dump.sh

```diff
@@ -16,10 +24,10 @@ dump() {
   log "Dumping certificates"
   traefik-certs-dumper file \
     --version v2 \
-    --crt-name "cert" \
-    --crt-ext ".pem" \
-    --key-name "key" \
-    --key-ext ".pem" \
+    --crt-name ${certificate_file_name} \
+    --crt-ext ${certificate_file_ext} \
+    --key-name ${privatekey_file_name} \
+    --key-ext ${privatekey_file_ext} \
     --domain-subdir \
     --dest /tmp/work \
     --source ${acme_file_path} >/dev/null
```

Comment:
```suggestion
    --source "${acme_file_path}" >/dev/null
```

Same applies here as of above.

Reply:
I see that this is recurring problem throughout the script. This should be fixed later on, but we can at least start applying those changes. :)